packages/site/src/main.css

```diff
@@ -133,7 +133,7 @@ iframe {
 .headerBlock {
   background-image: url("/img_collage.jpg");
   background-size: cover;
-  padding: var(--space-extravagant);
+  padding: var(--space-large);
```

Comment:
reduce padding on smaller viewports to reduce wrapping and match the padding of the collection container